http.go

```diff
@@ -83,40 +83,90 @@ type StdResponseWriter interface {
 	WriteString(data string) (int, error)
 }
 
-type logResponseWriter struct {
+// StdResponseWriter2 is the interface implemented by
+// the ResponseWriter from http.Server for HTTP/2 requests.
+//
+// HTTPServer's ResponseWriter implements this as well.
+type StdResponseWriter2 interface {
+	http.ResponseWriter
+	http.Flusher
+	http.CloseNotifier
+	http.Pusher
+	WriteString(data string) (int, error)
+}
+
+type logResponseWriter1 struct {
```

Comment:
This is just a question of style/preference, but how about keeping the `logResponseWriter` name? This would be more consistent with `StdResponseWriter/StdResponseWriter2`.

Reply:
Will fix.